src/foam/u2/dialog/ApplicationPopup.js

```diff
@@ -0,0 +1,154 @@
+foam.CLASS({
+  package: 'foam.u2.dialog',
+  name: 'ApplicationPopup',
+  extends: 'foam.u2.dialog.Popup',
+  documentation: `
+    A full-featured popup with the application's branding on it.
+  `,
+
+  implements: ['foam.mlang.Expressions'],
+
+  imports: [
+    'theme'
+  ],
+
+  exports: [
+    'close as closeDialog',
```

Comment:
Already exported in Popup

Reply:
oh right - I'm gonna change the export to `actionProvider` and move it into ApplicationPopup - I forgot to do that

Reply:
also what? I don't see that in my code locally